### DISCUSSION TITLE
Фиксировать вверху экрана заголовок таблицы ресурса при скролле

### DISCUSSION THREAD

**Discussion by shevl-dev** [moonshine-software/moonshine#1074] [Ideas]
Кажется было круто, особенно если в ресурсе очень много строк

**Comment by DissNik** [comment]
Какие идеи по реализации у вас есть?

Я пока не представляю как реализовать это через sticky:
- у таблиц есть горизонтальный скролл, sticky не будет работать
- есть компонент TopBar он находится на разных слоях с таблицей и фиксированные элементы не будут друг под другом

Если попытаться подключить какую-нибудь js библиотеку, то там скорее всего будет абсолютное позиционирование которое сломает соотношение ячеек таблицы с шапкой, нужно будет решать эту проблему и синхронизацию скролла контента таблицы с фиксированной шапкой

Еще можно сделать внутренний скролл в таблице, это не сложно и я сделал пример
https://github.com/moonshine-software/moonshine/pull/1075

**Reply by DissNik** [reply]
минус такого подхода в том что у таблицы свой скролл и нам нужно задать максимальную высоту
